compiler/index.js

```diff
@@ -32,6 +51,12 @@ exports.compile = function (text, options) {
     return 1 + Math.floor(Math.random() * max)
   }
 
+  var matcher = matchers.bms
+
+  if (options.format === 'dtx') {
+    matcher = matchers.dtx
```

Comment:
To avoid reassigning more variables than necessary, I recommend compacting these four lines into one:

```js
  var matcher = matchers[options.format] || matchers.bms
```